src/OpenLoco/src/ViewportManager.h

```diff
@@ -9,13 +9,14 @@
 
 namespace OpenLoco::Ui::ViewportManager
 {
+    constexpr size_t kMaxViewports = 256;
```

Comment:
Is this perhaps a bit overkill? I guess it's never that much memory we are talking about.